src/config/index.js

```diff
@@ -0,0 +1,13 @@
+import pool from './config';
+
+export default {
+  query(text, params) {
+    return new Promise((resolve, reject) => {

```

Comment:
Why are we wrapping a promise in a `new Promise` again?

Reply:
Thanks.. Resolved